### DISCUSSION TITLE
user-guide/add-items

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#267] [文档评论区]
# user-guide/add-items

Zotero 非官方中文维护小组

https://zotero-chinese.com/user-guide/add-items

**Comment by AlphaMind123** [comment]
期刊文章的图标上有两个人或者一支笔的那种是什么意思呢？

**Reply by baolongguaishou** [reply]
好像是预印本的意思喔

**Comment by mengwoo** [comment]
请问抓取条目后，为什么条目有些内容为空（比如期刊系列标题），会被自动添加中文的“无”，导致生成参考文献表的时候，中英文混杂，还有“无”的出现。比如：S A, A., C, S., P, D., G, A., Maniyan Lathikakumari, A., V Thomas, S., N Menon, R., 2024. Analysis of EEG microstates as biomarkers in neuropsychological processes - Review. Comput Biol Med, 无 173, 108266. https://doi.org/10.1016/j.compbiomed.2024.108266

**Comment by archwaynewu-alt** [comment]
求助：人在国外，茉莉花抓取知网文章一直在转圈，怎么解决呢？
